riscv/src/continuations/memory_merkle_tree.rs

```diff
@@ -1,7 +1,7 @@
 use std::collections::BTreeMap;
 
 use crate::bootloader::{
-    BYTES_PER_WORD, MERKLE_TREE_DEPTH as MERKLE_TREE_DEPTH_BOOTLOADER,
+    BYTES_PER_WORD, N_LEAVES_LOG as N_LEAVES_LOG_BOOTLOADER,
```

Comment:
I renamed `MERKLE_TREE_DEPTH` to `N_LEAVES_LOG`. Basically now:
- `N_LEAVES_LOG` is 22, because we have $2^{22}$ leaves
- `MERKLE_TREE_DEPTH` is now 23 (used to be 22), because we have levels 0-22 of $2^i$ nodes each